rest_api/common/src/paging.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright 2018-2022 Cargill Incorporated
```

Comment:
This should be versioned as v1 - see Grid sdk's rest_api::resources::paging::v1 module for layout.

Reply:
Added v1 module to paging